.github/workflows/basic.yml

```diff
@@ -31,7 +31,7 @@ on:
         type: string
       unit_runs_on:
         description: the runner group used for unit jobs run on
-        default: ubuntu-latest
+        default: ubuntu-24.04
```

Comment:
I would like to create a v3 branch based on v2. and on v3 we switch to ubuntu 24.04

Reply:
Sounds good! Do you want to run with creating v3, then I can rebase to that when its ready? :)

Reply:
I created the v3 branch, please update your PR

Reply:
I updated the PR! I could have made it named v3 for you, I just didnt know if you had other changes for v3 you wanted :)